pages/pipelines/trigger_step.md.erb

```diff
@@ -127,7 +127,12 @@ _Optional_ `build` _attributes:_
     branch: "${BUILDKITE_BRANCH}"
 ```
 
-<div class="Docs__note">
-<p class="Docs__note__heading">Environment variable support</p>
-<p>Buildkite Agent 3.0 or above is required to use pipeline.yml environment variable substitution. See the <a href="/docs/agent/v2/cli-pipeline#environment-variable-substitution">pipeline command documentation</a> for more information.</p>
-</div>
+To set environment variables for the triggered build itself, use the `env` attribute:
```

Comment:
Could we reword this a little? I found there was a bit of mental overhead when reading 'for the triggered build itself'.

> To set environment variables on the build created by the trigger step, use the `env` attribute:

Reply:
Heaps better! Updated.